src/main/java/Duke.java

```diff
@@ -1,10 +1,27 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String line = "-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-";
+
+        System.out.println("Hello from\n" + logo + "\n How may I help you today? :)");
+
+        String nextInput = input.nextLine();
+        boolean bye = nextInput.equals("bye");
+
+        while (!bye) {
+            System.out.println(line + "\n" + nextInput + "\n" + line);
+            nextInput = input.nextLine();
+            bye = nextInput.equals("bye");
+        }
+
+        System.out.println("Bye! I'll see you again next time!");
+
```

Comment:
Good code, but could have placed it into a method in Duke class. This would be better for abstraction.